lib/core/test/unit/Cardano/Pool/MetricsSpec.hs

```diff
@@ -88,30 +131,21 @@ epochLength :: EpochLength
 epochLength = EpochLength 50
 
 instance Arbitrary (Hash tag) where
-    arbitrary = do
-        InfiniteList bytes _ <- arbitrary
-        return $ Hash $ BS.pack $ take 32 bytes
-    shrink x = [zeros | x /= zeros]
-      where
-        zeros = Hash $ BS.pack $ replicate 32 0
+    shrink _  = []
+    arbitrary = Hash . B8.pack
+        <$> vectorOf 8 (elements (['a'..'f'] ++ ['0'..'9']))
 
 instance Arbitrary Block where
      arbitrary = genericArbitrary
      shrink = genericShrink
 
 instance Arbitrary (Quantity "block" Word32) where
-     arbitrary = Quantity . fromIntegral <$> (arbitrary @Word)
-
-instance Arbitrary (Quantity "lovelace" Word64) where
-     arbitrary = Quantity . fromIntegral <$> (arbitrary @Word64)
+     arbitrary = Quantity . fromIntegral <$> (arbitrary @Word32)
 
-instance Arbitrary (Quantity "block" Natural) where
+instance Arbitrary (Quantity any Word64) where
      arbitrary = Quantity . fromIntegral <$> (arbitrary @Word64)
 
 instance Arbitrary PoolId where
-    arbitrary = do
-        InfiniteList bytes _ <- arbitrary
-        return $ PoolId $ BS.pack $ take 32 bytes
-    shrink x = [zeros | x /= zeros]
-      where
-        zeros = PoolId $ BS.pack $ replicate 32 0
+    shrink _  = []
+    arbitrary = PoolId . B8.pack
+        <$> elements [ "ares", "athena", "hades", "hestia", "nemesis" ]
```

Comment:
I've used a much smaller set of pool ids, this makes the tests able to generate more interesting cases with less "empty combination".

Reply:
```
  combineMetrics
    pools with no entry for productions are included
      +++ OK, passed 100 tests.
    it fails if a block-producer is not in the stake distr
      +++ OK, passed 200 tests:
      71.5% Successfully combined the maps
      28.5% A pool without stake produced
  calculatePerformances
    performances are always between 0 and 1
      +++ OK, passed 100 tests (12% all null).
```